Flutter/xlsio/worksheet-rows-and-columns-manipulation.md

```diff
@@ -183,6 +183,53 @@ workbook.dispose();
 
 {% endhighlight %}
 
+## Show or Hide Rows and Columns
 
+Visibility of rows and columns can be set by using the showRows and showColumns methods as shown below.
```

Comment:
Don't we need to use like this for technical words or APIs? - `showRows`

Reply:
Yes sir. We can highlight the APIs or API reference links. I have added the API reference links to all the 3 APIs